docs/deploy-etcd.md

```diff
@@ -29,3 +33,17 @@ spec:
 - This should bring an etcd cluster in the `compose` namespace.
 
 **Note: this cluster configuration is really naive and does does not use mutual TLS to authenticate application accessing the data. For enabling mutual TLS, please refer to https://github.com/coreos/etcd-operator** 
+
+### Option2: Create a secure and highly available ETCD cluster
+
+This requires a slightly more advanced template, and some tooling for generating TLS credentials
+We will start with the same yaml as in option 1. Then we will add some options to make it more robust
+- First, enable persistent storage. To do this, follow https://github.com/coreos/etcd-operator/blob/master/doc/user/spec_examples.md#custom-persistentvolumeclaim-definition.
+  - To list the persistent storage classes available in your cluster, run `kubectl get storageclass`
+- Then, set an anti-affinity rule to prefer running ETCD pods on different nodes: https://github.com/coreos/etcd-operator/blob/master/doc/user/spec_examples.md#three-member-cluster-with-preferred-anti-affinity-between-pods-and-nodes-place-pods-in-different-nodes-if-possible
```

Comment:
could we safely do this one in the basic yaml provided?

Reply:
With the "prefer" constraint, yes. I will do that.